relay/polkadot/src/weights/runtime_parachains_coretime.rs

```diff
@@ -46,15 +47,27 @@ use core::marker::PhantomData;
 /// Weight functions for `runtime_parachains::coretime`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> runtime_parachains::coretime::WeightInfo for WeightInfo<T> {
+	/// Storage: `OnDemand::Revenue` (r:1 w:1)
+	/// Proof: `OnDemand::Revenue` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `System::Account` (r:2 w:2)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// Storage: `Dmp::DeliveryFeeFactor` (r:1 w:0)
+	/// Proof: `Dmp::DeliveryFeeFactor` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `XcmPallet::SupportedVersion` (r:1 w:0)
+	/// Proof: `XcmPallet::SupportedVersion` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Dmp::DownwardMessageQueues` (r:1 w:1)
+	/// Proof: `Dmp::DownwardMessageQueues` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Dmp::DownwardMessageQueueHeads` (r:1 w:1)
+	/// Proof: `Dmp::DownwardMessageQueueHeads` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	fn request_revenue_at() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `2963`
-		//  Estimated: `6428`
-		// Minimum execution time: 36_613_000 picoseconds.
-		Weight::from_parts(37_637_000, 0)
-			.saturating_add(Weight::from_parts(0, 6428))
-			.saturating_add(T::DbWeight::get().reads(6))
-			.saturating_add(T::DbWeight::get().writes(3))
+		//  Measured:  `2848`
+		//  Estimated: `6313`
+		// Minimum execution time: 92_241_000 picoseconds.
+		Weight::from_parts(93_731_000, 0)
+			.saturating_add(Weight::from_parts(0, 6313))
+			.saturating_add(T::DbWeight::get().reads(7))
+			.saturating_add(T::DbWeight::get().writes(5))
```

Comment:
@eskimor I now think you were right all along, I'm going to increase all the buffers on the coretime side. The 5% buffer I have right now makes me uncomfortable with possible increases as big as this (multiple additional writes) from one release to the next, we could get into a scenario where relay or coretime chain needs to be updated before the other. I'm going to just slap a 100% buffer on every hardcoded weight.

I still think the relay side weights are fine, as the largest ones only happen inside the migration so the versions are "locked" together at that stage and when both chains are the same version we have no problem (emulated tests ensure this).

For the ones that are sent regularly we should ensure that the buffers are big enough to allow big jumps like this in the case that one chain upgrades before the other so we don't have to rely on co-ordinated releases.